app/build.gradle

```diff
@@ -59,4 +61,7 @@ dependencies {
         exclude module: 'stax-api'
         exclude module: 'xpp3'
     }
+
+    compile 'com.github.bumptech.glide:glide:3.6.0'
```

Comment:
License OK: [BSD 2-Clause and Apache](https://github.com/bumptech/glide/blob/master/LICENSE)